tests/permissions.spec.ts

```diff
@@ -0,0 +1,392 @@
+import { test, expect, LxdVersions } from "./fixtures/lxd-test";
+import {
+  toggleIdentities,
+  addPermission,
+  createGroup,
+  deleteGroup,
+  editGroup,
+  randomGroupName,
+  removePermission,
+  restorePermission,
+  selectGroupAction,
+  visitGroupsPage,
+  selectGroups,
+} from "./helpers/auth-groups";
+import {
+  identityBar,
+  identityFoo,
+  selectIdentities,
+  toggleGroups,
+  visitIdentitiesPage,
+} from "./helpers/auth-identities";
+import {
+  createIdpGroup,
+  deleteIdpGroup,
+  editIdpGroup,
+  randomIdpGroupName,
+  visitIdpGroupsPage,
+} from "./helpers/auth-idp-groups";
+import {
+  assertModificationStatus,
+  confirmGroupsModifiedForIdentity,
+  confirmIdentitiesModifiedForGroup,
+  redoChange,
+  undoChange,
+} from "./helpers/auth";
+import {
+  resetSetting,
+  updateSetting,
+  visitServerSettings,
+} from "./helpers/server";
+
+const skipPermissionTest = (lxdVersion: LxdVersions) =>
+  test.skip(
+    lxdVersion !== "5.0-edge",
+    "Fine grained authorisation is not available for lxd 5.0",
+  );
+
+test("LXD 5.0 does not support permissions", async ({ page, lxdVersion }) => {
+  test.skip(
+    lxdVersion !== "5.0-edge",
+    "Fine grained authorisation is supported in lxd 5.21 LTS and latest/edge",
+  );
+  await page.goto("/ui/");
+  await expect(page.getByRole("button", { name: "Permissions" })).toBeHidden();
+});
+
+test.describe("Permission groups", () => {
+  test("create and delete group", async ({ page, lxdVersion }) => {
+    skipPermissionTest(lxdVersion);
+    const group = randomGroupName();
+    await visitGroupsPage(page);
+    await createGroup(page, group, `${group}-desc`);
+    await deleteGroup(page, group);
+  });
+
+  test("edit and rename group", async ({ page, lxdVersion }) => {
+    skipPermissionTest(lxdVersion);
+    const group = randomGroupName();
+    const renameGroup = randomGroupName();
+    await createGroup(page, group, group);
+    await editGroup(page, group, renameGroup, renameGroup);
+    await editGroup(page, renameGroup, group, group);
```

Comment:
We seem to never read the description, shall we remove description parameters from the function arguments for create and edit group?
Can we call "edit group" as `renameGroup`, because I think that's what it does?